app/scripts/store/dimStoreHeading.directive.js

```diff
@@ -55,6 +55,18 @@
         });
       });
 
+      vm.getLevelBar = function getLevelBar() {
+        if(vm.store.percentToNextLevel) {
+          return {width: vm.store.percentToNextLevel + '%'};
+        }
+        if(vm.store.progression && vm.store.progression.progressions) {
+          vm.xpTillMote = 'Prestige level: ' + vm.store.progression.progressions[4].level + '\n' +
```

Comment:
good call. meant to get back to that. i'll use underscore and pluck via the hash